appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/admin/OpenApiServiceConfiguration.java

```diff
@@ -60,22 +60,28 @@
 
     /**
      * Defines if CORS headers are set on the OpenApi response.
-     * 
+     *
      * @return whether to set CORS headers or not
      */
     @Attribute(defaultValue = "false", dataType = Boolean.class)
     String getCorsHeaders();
 
     void setCorsHeaders(String value) throws PropertyVetoException;
-    
+
     /**
-     * String value defines the attached virtual servers.
-     *
-     * @return
+     * @return a String value defines the attached virtual servers.
      */
     @Attribute(dataType = String.class)
     String getVirtualServers();
 
     void setVirtualServers(String value) throws PropertyVetoException;
 
+    /**
+     * @return a Boolean value determining if the security is enabled or not.
+     */
+    @Attribute(defaultValue = "false", dataType = Boolean.class)
+    String getSecurityEnabled();
+
+    void setSecurityEnabled(String value) throws PropertyVetoException;
```

Comment:
Rename get and set methods to `setAuthorizationRequired` and `getAuthorizationRequired`